scripts/automation/trex_control_plane/stl/trex_stl_lib/trex_stl_hltapi.py

```diff
@@ -424,15 +441,84 @@ def cleanup_session(self, **user_kwargs):
         self.trex_client = None
         return HLT_OK()
 
-    def interface_config(self, port_handle, mode='config'):
+    def interface_config(self, **user_kwargs):
         if not self.trex_client:
             return HLT_ERR('Connect first')
+
+        kwargs = merge_kwargs(interface_config_kwargs, user_kwargs)
+        is_array_based = not isinstance(kwargs['port_handle'], int)
```

Comment:
Please reconsider this check, port handle can be for example string "1" or "1 2".
Look at function _parse_port_list